### DISCUSSION TITLE
Help, My account has been hacked by someone !

### DISCUSSION THREAD

**Discussion by dit0day** [community/community#114200] [Other Feature Feedback, Questions, & Ideas] [answered]
I just received an email notification that my GitHub account has changed its password and email. Even though I swore I didn't do that at all, so I thought there was something wrong with this and someone was trying to take over my account. I ask for help to resolve this and help restore my account. I am also ready to provide some necessary evidence to prove that the account is really mine.

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @bruce ! . You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
